crates/handlers/src/oauth2/introspection.rs

```diff
@@ -625,6 +625,11 @@ pub(crate) async fn post(
                 device_id: session.device.map(Device::into),
             }
         }
+
+        TokenType::PersonalAccessToken => {
+            // TODO
+            return Err(RouteError::UnknownToken(TokenType::PersonalAccessToken));
```

Comment:
A few things I have in mind: this needs to check that both the actor and the owner are still active.

Deactivating (not locking) users should revoke all their tokens

Validating those tokens should also be done for requests to the GraphQL endpoint (optional maybe, we want to deprecate 'admin' access to the GraphQL API) and for the admin API access